.devcontainer/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/rust/.devcontainer/base.Dockerfile
+
+# [Choice] Debian OS version (use bullseye on local arm64/Apple Silicon): buster, bullseye
+ARG VARIANT="buster"
```

Comment:
bullseye is the later OS version, in general we prefer later versions to earlier ones.

This is also inconsistent with devcontainer.json, which uses bullseye.